modules/oe_content_project/config/install/field.field.node.oe_project.oe_project_funding_programme.yml

```diff
@@ -0,0 +1,34 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_project_funding_programme
+    - node.type.oe_project
+  module:
+    - rdf_skos
+id: node.oe_project.oe_project_funding_programme
+field_name: oe_project_funding_programme
+entity_type: node
+bundle: oe_project
+label: 'Funding Programme'
```

Comment:
The field labels in content types are first upper case then lower case like: Funding programme
I know the ticket said otherwise, but I would keep consistency.

Reply:
makes sense yes

Reply:
Config and Test changed